common/Helpers/ManagementInfrastructureHelper.cs

```diff
@@ -64,4 +71,20 @@ private static FeatureAvailabilityKind GetAvailabilityKindFromState(uint state)
                 return FeatureAvailabilityKind.Unknown;
         }
     }
+
+    /// <summary>
+    /// Gets a boolean indicating whether the extension should be disabled if the Windows optional
+    /// feature is absent from the machine.
+    /// </summary>
+    /// <param name="extensionClassId">The class Id of the out of proc extension object</param>
+    /// <returns>True when the Windows optional feature is absent. False otherwise.</returns>
+    public static bool ShouldDisableExtensionIfFeatureAbsent(string extensionClassId)
```

Comment:
NIT:  Change to IsExtensionEnabled.  This changes the method from a negative to a positive.  if(IsExtensionEnabled) is simpler to read than if(!ShouldDisableExtension).  One less negative to think about.